test/expected/truncate_hypertable.out

```diff
@@ -0,0 +1,134 @@
+NOTICE:  index "5-two_Partitions_timeCustom_series_bool_idx" does not exist, skipping
+NOTICE:  index "6-two_Partitions_timeCustom_device_id_idx" does not exist, skipping
+NOTICE:  index "7-two_Partitions_timeCustom_idx" does not exist, skipping
+NOTICE:  index "10-two_Partitions_timeCustom_series_1_idx" does not exist, skipping
+NOTICE:  index "11-two_Partitions_timeCustom_series_2_idx" does not exist, skipping
+NOTICE:  index "12-two_Partitions_timeCustom_series_bool_idx" does not exist, skipping
+NOTICE:  index "13-two_Partitions_timeCustom_device_id_idx" does not exist, skipping
+NOTICE:  index "14-two_Partitions_timeCustom_idx" does not exist, skipping
+NOTICE:  index "8-two_Partitions_device_id_timeCustom_idx" does not exist, skipping
+NOTICE:  index "9-two_Partitions_timeCustom_series_0_idx" does not exist, skipping
+NOTICE:  index "15-two_Partitions_device_id_timeCustom_idx" does not exist, skipping
+NOTICE:  index "16-two_Partitions_timeCustom_series_0_idx" does not exist, skipping
+NOTICE:  index "17-two_Partitions_timeCustom_series_1_idx" does not exist, skipping
+NOTICE:  index "18-two_Partitions_timeCustom_series_2_idx" does not exist, skipping
+NOTICE:  index "19-two_Partitions_timeCustom_series_bool_idx" does not exist, skipping
+NOTICE:  index "20-two_Partitions_timeCustom_device_id_idx" does not exist, skipping
+NOTICE:  index "21-two_Partitions_timeCustom_idx" does not exist, skipping
+NOTICE:  index "22-two_Partitions_device_id_timeCustom_idx" does not exist, skipping
+NOTICE:  index "23-two_Partitions_timeCustom_series_0_idx" does not exist, skipping
+NOTICE:  index "24-two_Partitions_timeCustom_series_1_idx" does not exist, skipping
+NOTICE:  index "25-two_Partitions_timeCustom_series_2_idx" does not exist, skipping
+NOTICE:  index "26-two_Partitions_timeCustom_series_bool_idx" does not exist, skipping
+NOTICE:  index "27-two_Partitions_timeCustom_device_id_idx" does not exist, skipping
+NOTICE:  index "28-two_Partitions_timeCustom_idx" does not exist, skipping
+SELECT * FROM _timescaledb_catalog.hypertable;
+ id | schema_name |   table_name   | associated_schema_name | associated_table_prefix | num_dimensions 
+----+-------------+----------------+------------------------+-------------------------+----------------
+  1 | public      | two_Partitions | _timescaledb_internal  | _hyper_1                |              2
+(1 row)
+
+SELECT * FROM _timescaledb_catalog.chunk;
+ id | hypertable_id | schema_name | table_name 
+----+---------------+-------------+------------
+(0 rows)
+
+-- should be empty
+\set ON_ERROR_STOP 0
+\dt "_timescaledb_internal".*
+      List of relations
+ Schema | Name | Type | Owner 
+--------+------+------+-------
+(0 rows)
+
+\set ON_ERROR_STOP 1
+SELECT * FROM "two_Partitions";
+ timeCustom | device_id | series_0 | series_1 | series_2 | series_bool 
+------------+-----------+----------+----------+----------+-------------
+(0 rows)
+
```

Comment:
Can we also add a `\d+ "two_Partitions"` to check that it has no inherited children